pennylane/utils.py

```diff
@@ -417,3 +417,57 @@ def expand_vector(vector, original_wires, expanded_wires):
     )
 
     return qml.math.reshape(expanded_tensor, 2 ** M)
+
+
+def get_generator(op, return_matrix=False):
```

Comment:
this utility function essentially abstracts away (for now) the differences between the different operators.